js/index.js

```diff
@@ -71,3 +71,23 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
   }
 });
+
+let formData = {
+  name: '',
+  email: '',
+  message: '',
+};
+
+if (localStorage.getItem('formData') !== null) {
+  const data = localStorage.getItem('formData');
+  formData = JSON.parse(data);
+}
+
+const formElements = document.querySelectorAll('input', 'area');
+formElements.forEach((element) => {
+  element.value = formData[element.name];
+  element.addEventListener('input', (e) => {
+    formData[e.target.name] = e.target.value;
+    localStorage.setItem('formData', JSON.stringify(formData));
+  });
+});
```

Comment:
- [ ] Great job so far, I tried searching your slack id to contact you but couldn't find it. Your name and email are working correctly with persistent data. however, your message is not persistent upon refreshed. Kindly refer below for the same and fix it.

https://user-images.githubusercontent.com/70269507/174439572-fedc06fc-2e03-416f-980e-d95df0327aa2.mov

Reply:
@sja-thedude 
slack name: Muhyideen Elias

Reply:
@sja-thedude
email : emuhyideen6@gmail.com